autoware.repos

```diff
@@ -33,7 +33,7 @@ repositories:
   core/autoware.core:
     type: git
     url: https://github.com/autowarefoundation/autoware.core.git
-    version: main
+    version: df9cd361d74f61de5aa7b800e5fba576755ac3eb
```

Comment:
```suggestion
    version: v0.0.0
```

Reply:
Since other core packages, as well as `autoware.universe` and `autoware_launch`, manage versioning with the standard `*.*.*` tag notation used by ROS release tools, we would like you to use `*.*.*` instead of `v*.*.*`.
https://docs.ros.org/en/humble/How-To-Guides/Releasing/First-Time-Release.html#bump-the-package-version

Reply:
I have pushed the tag as 0.0.0 and modified the autoware.repos accordingly.